docs/Manual/09 Parameters.md

```diff
@@ -80,5 +80,4 @@ package.add(nest, args=(A, B, C), base_name="matmul", parameters)
 ```
 In this case, `package.add` generates a function eight times, once for each parameter combination in the grid.  Other than `nest`, `package.add` can alternatively accept a `Schedule` (if we are performing schedule transformations), or a `Plan` (if we are setting target-specific options). All eight functions share the same base name. However, Accera automatically adds a unique suffix to each function name to prevent duplication. This pattern allows optional filtering by inspecting the generated parameter values list before calling `package.add`.
```

Comment:
In case you're wondering, I fixed the merge conflicts for Parameters.md and Packages.md so these show up here, but you don't have to do anything with them.